packages/core/src/util/array.ts

```diff
@@ -0,0 +1,16 @@
+const move = function(array: Array<any>, index: number, delta: number) {
+  const newIndex: number = index + delta;
+  if (newIndex < 0 || newIndex >= array.length) return; //Already at the top or bottom.
+  var indexes: number[] = [index, newIndex].sort(); //Sort the indixes
+  array.splice(indexes[0], 2, array[indexes[1]], array[indexes[0]]);
+};
+
+var moveUp = function(array: Array<any>, toMove: number) {
```

Comment:
I suggest to use `const` instead of `var` and Fat Arrows instead of function. 
ES Lint should complain, but something is not working as expected there